crates/rethnet_evm/benches/state.rs

```diff
@@ -16,10 +19,27 @@ impl RethnetStates {
         let mut states: [&mut dyn SyncState<StateError>; 2] = [&mut self.layered, &mut self.hybrid];
         for state in states.iter_mut() {
             let number_of_checkpoints = number_of_accounts / number_of_accounts_per_checkpoint;
-            for _ in 0..=number_of_checkpoints {
-                for i in 1..=number_of_accounts_per_checkpoint {
+            for checkpoint_number in 0..=number_of_checkpoints {
```

Comment:
```suggestion
            for checkpoint_number in 0..number_of_checkpoints {
```
This should be an exclusive range, as we're starting at 0